godot/etc/Globals.cs

```diff
@@ -2,4 +2,6 @@
 {
     public const float c_unitsToMeters = 10f;
     public const float c_metersToUnits = 1f / c_unitsToMeters;
+    public const float c_baseGameDuration = 60f;
```

Comment:
maybe out of scope, but it might make sense to make this data driven and fill in these values from the editor

Reply:
Will make this change in subsequent PR